### PR TITLE
Cleanup Gradle configuration files

### PR DESCRIPTION
**Pull request by arturdryomov**
Just some blank lines and repositories declaration.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,14 +1,14 @@
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```
Comment (arturdryomov) [inline build.gradle]: JCenter supersedes Maven Central.

File: build.gradle [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
         maven {
             url 'https://dl.bintray.com/kotlin/kotlin-eap'
         }
         maven {
             url 'http://oss.sonatype.org/content/repositories/snapshots'
```
Comment (arturdryomov) [inline build.gradle]: Do we actually needs these two at this point?

Comment (artem-zinnatullin) [reply build.gradle]: Let's try to remove them, I hope we don't use snapshots nor eap for public releases

**Review by artem-zinnatullin — COMMENTED** [review]
// I don't see '.' at the end of commit message :trollface:

**Review by artem-zinnatullin — APPROVED** [review]

**Comment by artem-zinnatullin** [comment]
LGTM, can you please resolve conflict?

**Comment by arturdryomov** [comment]
Done via GitHub (works like a miracle after Bitbucket).

**Comment by artem-zinnatullin** [comment]
New conflict :)

**Comment by artem-zinnatullin** [comment]
I think we broke CI after migrating org on github

**Comment by arturdryomov** [comment]
DONE AGAIN

**Comment by artem-zinnatullin** [comment]
RELAX MAN, I MERGED IT ALREADY, JEEZ

**Pull request merged**